Sources/SwiftSyntax/generated/syntaxNodes/SyntaxNodesD.swift

```diff
@@ -636,6 +678,14 @@ public struct DeclReferenceExprSyntax: ExprSyntaxProtocol, SyntaxHashable, _Leaf
     }
   }
   
+  /// For syntax trees generated by the parser, this is guaranteed to be one of the following token kinds:
+  ///  - `<identifier>`
```

Comment:
Mmmm, backticks are not needed — I'll remove them in the `GrammarGenerator` for this.